server/startup/load-data.js

```diff
@@ -20,13 +20,12 @@ export default function () {
       } catch (error) {
         Logger.error(error, "Bypassing loading Shop default data");
       }
-    }
-
 
-    // make sure the default shop has been created before going further
-    while (!Reaction.getShopId()) {
-      Logger.debug("Loading default shop, waiting until it's ready before moving on...");
-      Meteor._sleepForMs(1000);
+      // make sure the default shop has been created before going further
+      while (!Reaction.getShopId()) {
+        Logger.debug("Loading default shop, waiting until it's ready before moving on...");
+        Meteor._sleepForMs(1000);
+      }
```

Comment:
Moved this `while` to be inside the `if` because there is no need to check `Reaction.getShopId()` again right after already checking it in the `if` condition